### PR TITLE
Add haemoglobin codes v1

### PR DESCRIPTION
**Pull request by maiparkes58**
I want to try with haemoglobin codes first to know the procedure of adding new code sets to the shared folder. Please comment anything I should note to make it easier and efficient in the upcoming steps.

### CODE REVIEW

**Comment by Georgetilston** [comment]
A few comments from me:
The folder structure needs to be the same as other code sets. Currently there is no folder indicating the version number for the code set (which in this case will be '1').
Also, not all of those haemoglobin codes will be needed for this study. RQ34 is only looking at certain values for hb, so we will only need codes that give us a specific measurement, or codes that align with the criteria in the study proposal. If we can find out what the accepted range is for codes like 'haemoglobin - very high' then we can decide whether to include any of these codes. I would suggest investigating each code in GP_Events to decide which ones are needed for the purposes of this study, and looking onlyine for any definitions of codes like 'very high'.

**Review by rw251 — COMMENTED** [review]
Thanks for changing to the correct folder structure. There is actually documentation about this here: https://github.com/rw251/gm-idcr/blob/master/docs/SQL-generation-process.md.

As well as folder structure, versioning, and file naming conventions, it also shows that you can create the outline structure of a clinical code set by running the `run - windows.bat` file and following on-screen instructions.

You'll also need to add a README.md file. See the other code-sets for an example.

Finally, if there is anything that is unclear, or not mentioned in the documentation, please make a pull request for that as well. Thanks!

**Review by Georgetilston — APPROVED** [review]
looks good to me, as long as all these codes have associated values in GP_Events

**Comment by maiparkes58** [comment]
Yeah they all have values with haemoglobin units. Thanks both for the comments.

**Review by Georgetilston — COMMENTED** [review]
The README states that the code set 'does not include codes like 'Haemoglobin abnormal' but it does include a code for that. Could you either remove the code if it is not needed, or edit the README?

**Comment by maiparkes58** [comment]
Thanks George for noticing. Yes it is more accurate to replace it with the code we excluded.

File: shared/clinical-code-sets/tests/haemoglobin/1/README.md [hunk 1]
```diff
@@ -0,0 +1,17 @@
+# Haemoglobin
+
+A patient's haemoglobin as recorded via clinical code and value. This code set only includes codes that are accompanied by a value (`423.. - Haemoglobin estimation`). It does not include codes that indicate a patient's haemoglobin (`4235 - Haemoglobin low`) without giving the actual value.
+
+Haemoglobin codes retrieved from https://www.medrxiv.org/content/medrxiv/suppl/2020/05/19/2020.05.14.20101626.DC1/2020.05.14.20101626-1.pdf and metadata available in this directory.
```
Comment (rw251) [inline shared/clinical-code-sets/tests/haemoglobin/1/README.md:5]: Similar to my comment in the ferritin pull request... When we talk about metadata for a code set, we don't mean the list of codes - instead we mean the description of how the set was constructed. Typically this is json files downloaded from GetSet. So just remove the bit about metadata from this line.

**Review by rw251 — APPROVED** [review]
Once the reference to metadata is removed from the README then this is ready to merge.

**Pull request merged**